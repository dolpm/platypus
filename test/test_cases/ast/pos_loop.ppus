pipe main [] |> unit {
    loop 0 -> 5 as i {
        Printnl <| [int_to_string <| [i]]; 
    }

    loop 0 -> 5 as (i, 2) {
        Printnl <| [int_to_string <| [i]];
    }

    while (i < 10) {
        Printnl <| [int_to_string <| [i]];
        i = (i + 1);
    }

    |> ();
}