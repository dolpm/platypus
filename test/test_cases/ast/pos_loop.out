pipe main |> [] |> [] |> unit {
  loop 0 -> 5 as (i,1)  {
    printnl <| [int_to_string <| [i]];
  }
  loop 0 -> 5 as (i,2)  {
    printnl <| [int_to_string <| [i]];
  }
  while i < 10
  {
    printnl <| [int_to_string <| [i]];
    i <| i + 1;
  }
  |> ();
}
